course/src/main/java/org/openedx/course/presentation/unit/html/HtmlUnitFragment.kt

```diff
@@ -210,14 +211,12 @@ private fun HTMLContentView(
                         onWebPageLoaded()
 
                         evaluateJavascript(
```

Comment:
Could we please include a one-liner documentation in this section for JS, briefly explaining its purpose for other developers? 

Additionally, could we add detailed documentation in the JavaScript file specifying the XBlocks we are affecting with this code?

Reply:
done